std_msgs/Header header
geometry_msgs/Pose pose
bool gripper_open
bool move_enabled
